tests/simulation/seeding/test_seeding.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+
+from simba_ml.simulation import distributions
+from simba_ml.simulation import species
+from simba_ml.simulation import kinetic_parameters as kinetic_parameters_module
+from simba_ml.simulation import noisers
+from simba_ml.simulation import system_model
+from simba_ml.simulation import random_generator
+from simba_ml.simulation import generators
+
+
+def list_of_dataframes_similar(list1, list2, max_difference=0.00001):
+    if not len(list1) == len(list2):
+        return False
+    for i in range(len(list1)):
+        if not (
+            list1[i].to_numpy().reshape(-1) - list2[i].to_numpy().reshape(-1)
+            <= max_difference
+        ).all():
+            return False
+    return True
+
+
+def test_seeding():
+    sm = create_example_system_model()
+    generator = generators.TimeSeriesGenerator(sm)
+    random_generator.set_seed(42)
+    result = generator.generate_signals(2)
+    expected_result = [
+        pd.DataFrame(
+            [
+                [9.15250908],
+                [7.37045458],
+                [5.54378722],
+                [8.88790029],
+                [6.5422931],
+                [7.54977531],
+                [5.26495249],
+                [5.22911248],
+                [6.61273201],
+                [14.05146558],
+            ],
+            columns=["y"],
+        ),
+        pd.DataFrame(
+            [
+                [x]
+                for x in [
+                    4.981558,
+                    16.183999,
+                    3.526283,
+                    4.634122,
+                    7.221429,
+                    7.757346,
+                    14.852047,
+                    9.979742,
+                    6.312557,
+                    16.227660,
+                ]
+            ],
+            columns=["y"],
+        ),
+    ]
```

Comment:
I dunno how nitty picky we want to be with our tests but we could (!) be consistent with the dataframe definitions. It's up to you to decide whether you change the second dataframe to match the first.

Reply:
Consitified.